shared/runtime/pyexec.c

```diff
@@ -720,6 +720,11 @@ int pyexec_frozen_module(const char *name, bool allow_keyboard_interrupt) {
 }
 #endif
 
+int pyexec_str(vstr_t *str, bool allow_keyboard_interrupt) {
```

Comment:
I suggest calling it `pyexec_vstr`, in case one day we do add a function that takes a null-terminated string (although that would probably be called `pyexec_cstr`).

Reply:
Renamed to `pyexec_vstr`